src/core/binding.c

```diff
@@ -720,6 +720,11 @@ QuicBindingQueueStatelessOperation(
     }
 
     QUIC_WORKER* Worker = QuicLibraryGetWorker(Datagram);
+    if (Worker == NULL) {
+        QuicPacketLogDrop(Binding, CxPlatDataPathRecvDataToRecvPacket(Datagram),
+            "Stateless worker non-existent (stateless oper)");
+        return FALSE;
+    }
```

Comment:
This shouldn't be possible. If `MsQuicLib.StatelessRegistration` is non-null, `QuicLibraryGetWorker` is required to succeed. `QuicLibraryGetWorker` just returns a ref to memory, so it can never be `NULL`, though, it's theoretically possible it's pointing to uninitialized memory if there is a bug somewhere else.
```c
    return
        &MsQuicLib.StatelessRegistration->WorkerPool->Workers[
            Datagram->PartitionIndex % MsQuicLib.PartitionCount];
```
If you encountered a problem here, then I suspect the real problem is that `MsQuicLib.StatelessRegistration` should have been cleaned up if it's initialization wasn't 100% successful.

Reply:
Ok, better fix incoming.